doc/ipynb_to_py.py

```diff
@@ -0,0 +1,43 @@
+"""Convert jupyter notebook to sphinx gallery notebook styled examples.
+
+Usage: python ipynb_to_gallery.py <notebook.ipynb>
+
+Dependencies:
+pypandoc: install using `pip install pypandoc`
+"""
+import pypandoc as pdoc
+import json
+
+
+def convert_ipynb_to_gallery(file_name):
```

Comment:
Something like this could indeed be a solution to for maintaining duplicate python and .ipynb examples, but I wonder if keeping the `.py` version and letting sphinx-gallery generate the `.ipynb` wouldn't be more robust in the long run (assuming that a workaround for widgets is found).

Reply:
Sure, hence my question about the 'typical flow' in this case.